kroxylicious-kms-provider-hashicorp-vault/src/main/java/io/kroxylicious/kms/provider/hashicorp/vault/VaultKms.java

```diff
@@ -186,4 +207,8 @@ else if (r.statusCode() != 200) {
             return handler.apply(r);
         };
     }
+
+    /* testing */ URI getEngineUri() {
```

Comment:
Nit: i think we have `@visbableForTesting` available

Reply:
The annotation is part of kroxylicious-core. It probably shouldn't be. https://github.com/kroxylicious/kroxylicious/issues/731

I don't want to fix this now.